source/javascripts/components/GuidedOnboarding/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useRef, useEffect } from "react";
+import { Icon, Text, Flex } from "@bitrise/bitkit";
+import "./Collapsible.scss";
+interface CollapsibleProps {
+  open?: boolean;
+  title: string;
+}
+
+const Collapsible: React.FC<CollapsibleProps> = ({ open = false, children, title }) => {
```

Comment:
I think the FE community here is aligned on not using FC for typing components.

A few reasons for why to avoid it: 
https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/
https://stackoverflow.com/questions/71788254/react-18-typescript-children-fc

Reply:
noted, will update